### ISSUE TITLE
error[E0658]: binding by-move and by-ref in the same pattern is unstable when compiling from source.

### ISSUE THREAD

**Issue by benalb** [ImageOptim/gifski#160]
In x86_64 with cargo 1.48.0 get this error:

```
 benalb@void-lite  ~/tmp/gifski   GIT  main
cargo build --release --features=openmp
    Updating crates.io index
(...)
   Compiling gifski v1.2.6 (/home/benalb/tmp/gifski)
error[E0658]: binding by-move and by-ref in the same pattern is unstable
  --> src/encoderust.rs:24:43
   |
24 |         let GIFFrame {left, top, ref pal, image, screen_width, screen_height, dispose} = f;
   |                                  -------  ^^^^^ by-move pattern here
   |                                  |
   |                                  by-ref pattern here
   |
   = note: see issue #68354 <https://github.com/rust-lang/rust/issues/68354> for more information

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: could not compile `gifski`
```

**Comment by kornelski** [comment]
The latest stable 1.49 supports this.

**Comment by kornelski** [comment]
fixed in 3b0b8275241e2df056edb7c431df6d7b5346758b
